<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hwmon</class>
 <widget class="QMainWindow" name="Hwmon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hwmon</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lab_lm75">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>LM75</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>61</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lab_lm75_tmp">
       <property name="text">
        <string>Temp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_lm75_high">
       <property name="text">
        <string>High</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_lm75_hyst">
       <property name="text">
        <string>Hyst</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>141</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="val_lm75_tmp">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="val_lm75_high">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="val_lm75_hyst">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>911</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lab_f75387">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>F75387</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>61</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lab_in0">
       <property name="text">
        <string>IN0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_in1">
       <property name="text">
        <string>IN1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_in2">
       <property name="text">
        <string>IN2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_in3">
       <property name="text">
        <string>IN3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_fan1">
       <property name="text">
        <string>FAN1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_fan2">
       <property name="text">
        <string>FAN2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_tmp1">
       <property name="text">
        <string>Temp1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_tmp2">
       <property name="text">
        <string>Temp2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lab_75387_val_in0">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_75387_val_in1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_75387_val_in2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_75387_val_in3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_75387_val_fan0">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_75387_val_fan1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_75387_val_temp0">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_75387_val_temp1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lab_soc_thermal">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SOC Thermal</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>16</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lab_thermal_temp0">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>76</width>
      <height>19</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_thermal_temp_val0">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>158</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_thermal0_type1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Critical</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="lab_thermal0_trig1">
       <property name="text">
        <string>Trigger</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_thermal0_hyst1">
       <property name="text">
        <string>Hyst</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>60</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="lab_thermal0_trig_val1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_thermal0_hyst_val1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lab_thermal_0">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>89</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Cpu-Thermal</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_thermal_1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SOC-Thermal</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_thermal1_type1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Critical</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_11">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>51</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="QLabel" name="lab_thermal1_trig1">
       <property name="text">
        <string>Tigger</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_thermal1_hyst1">
       <property name="text">
        <string>Hyst</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_12">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>190</y>
      <width>111</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <widget class="QLabel" name="lab_thermal1_trig_val1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_thermal1_hyst_val1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lab_thermal_temp1">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab_thermal_temp_val1">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
